dask/dataframe/reshape.py

```diff
@@ -220,14 +222,23 @@ def pivot_table(df, index=None, columns=None, values=None, aggfunc="mean"):
             "`df[columns].cat.as_known()` beforehand to ensure "
             "known categories"
         )
-    if not is_scalar(values) or values is None:
-        raise ValueError("'values' must be the name of an existing column")
+    if not (
+        is_list_like(values)
+        and all([is_scalar(v) for v in values])
+        or is_scalar(values)
+    ):
+        raise ValueError("'values' must be refer to existing an column or columns")
     if not is_scalar(aggfunc) or aggfunc not in ("mean", "sum", "count"):
         raise ValueError("aggfunc must be either 'mean', 'sum' or 'count'")
 
-    # _emulate can't work for empty data
-    # the result must have CategoricalIndex columns
-    new_columns = pd.CategoricalIndex(df[columns].cat.categories, name=columns)
+    columns_contents = pd.CategoricalIndex(df[columns].cat.categories, name=columns)
+    if is_scalar(values):
+        new_columns = columns_contents
+    else:
+        new_columns = pd.MultiIndex.from_product(
+            (values, columns_contents), names=[None, columns]
```

Comment:
I think the values should be sorted before passing off to `pd.MultiIndex` otherwise the order of the `._meta` won't match the order of the output. 

Compare:
```python
import pandas as pd
import dask.dataframe as ds
df = pd.DataFrame({
    'A':[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20],
    'B':['1th','2th','3th','4th','5th','1th','1th','3th','3th','3th','1th','2th','3th','4th','5th','1th','1th','3th','3th','3th'],
    'C':['aa','bb','cc','aa','cc','aa','bb','cc','aa','cc','aa','bb','cc','aa','cc','aa','bb','cc','aa','cc'],
    'D':[25,50,45,14,12,89,7,11,32,10,25,50,45,14,12,89,7,11,32,10]
})
df.pivot_table(index='B',columns='C',values=['D','A'])
```
to 
```python
dsdf = ds.from_pandas(df,npartitions=2)
dsdf['C'] = dsdf['C'].astype('category').cat.as_known()
dsdf.pivot_table(index='B',columns='C',values=['D','A'])
```

Reply:
Not sure I follow. What difference should i be looking for in those two snippets?

Reply:
Sorry shoul have included output :) If you execute them you'll see that in the pandas version the columns are sorted so the output is `A`, `D`:
```
        A                 D            
C      aa    bb    cc    aa    bb    cc
B                                      
1th   8.5  12.0   NaN  57.0   7.0   NaN
2th   NaN   7.0   NaN   NaN  50.0   NaN
3th  14.0   NaN  12.0  32.0   NaN  22.0
4th   9.0   NaN   NaN  14.0   NaN   NaN
5th   NaN   NaN  10.0   NaN   NaN  12.0
```
whereas in the dask one the columns are not sorted:

```
Dask DataFrame Structure:
                     D                          A                  
                    aa       bb       cc       aa       bb       cc
npartitions=1                                                      
               float64  float64  float64  float64  float64  float64
                   ...      ...      ...      ...      ...      ...
Dask Name: truediv, 17 tasks
```

So I am suggesting:
```suggestion
            (sorted(values), columns_contents), names=[None, columns]
```